scripts/migration/migrate_date_modified_for_existing_nodes.py

```diff
@@ -0,0 +1,37 @@
+"""
+This will add a date_modified field to all nodes.  Date_modified will be equivalent to the date of the last log.
+"""
+import sys
+import logging
+
+from website import models
+from website.app import init_app
+
+logger = logging.getLogger(__name__)
+
+def date_updated(self):
+        """
+        The most recent datetime when this node was modified, based on
+        the logs.
+        """
+        try:
+            return self.logs[-1].date
+        except IndexError:
+            return self.date_created
+
+def main():
+    init_app(routes=False)
+    dry_run = 'dry' in sys.argv
+    logger.warn('Date_modified field will be added to all nodes.')
+    if dry_run:
+        logger.warn('Dry_run mode')
+    for node in models.Node.find():
+        logger.info('Node {0} "date_modified" added'.format(node._id))
```

Comment:
Might make sense to put this logger statement after the node is saved so that it is only emitted after a successful migration of a single node.